tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -118,6 +119,7 @@ protected boolean isWriteActionRequired() {
   public void testStructTags()                { doTest(); }
   public void testContinue()                  { doTest(); }
   public void testEqualinif()                 { doTest(); }
+  public void testDeferInLoop()               { doTest(); }
```

Comment:
Updated the test to run in a different way. Thank you for pointing out the info about `doTest`